onnxruntime/test/python/quantization/test_op_reshape.py

```diff
@@ -76,12 +75,13 @@ def test_quantize_reshape(self):
                                   [3, 7],
                                   [7, 3],
                                   [1, 9])
-        quantize_static(model_fp32_path,
-                        model_uint8_path,
-                        data_reader
-                        )
+        quantize_static(model_fp32_path, model_uint8_path, data_reader)
         data_reader.rewind()
         qdq_nodes = {'QLinearMatMul': 1, 'QuantizeLinear': 1, 'DequantizeLinear': 1, 'Reshape': 1}
```

Comment:
>qdq_nodes [](start = 8, length = 9)

nit: name is not correct.

Reply:
And could you please add test for QDQ format

---
In reply to: [609102392](https://github.com/microsoft/onnxruntime/pull/7265#discussion_r609102392) [](ancestors = 609102392)

Reply:
cool